tensorflow/core/ops/parsing_ops.cc

```diff
@@ -27,7 +27,7 @@ using shape_inference::ShapeHandle;
 REGISTER_OP("DecodeRaw")
     .Input("bytes: string")
     .Output("output: out_type")
-    .Attr("out_type: {float,double,int32,uint8,int16,int8,int64}")
+    .Attr("out_type: {float16,float,double,int32,uint8,int16,int8,int64}")
```

Comment:
We call 'float16' 'half' instead in tensorflow. See for example the definition of the BatchMatMul op in tensorflow/core/ops/math_ops.cc